website/docs/provisioners/connection.html.markdown

```diff
@@ -77,6 +77,10 @@ provisioner "file" {
   [the `file` function](/docs/configuration/functions/file.html). This takes
   preference over the password if provided.
 
+* `certificate` - The contents of an signed CA Certificate. These can
+  be loaded from a file on disk using the [`file()` interpolation
+  function](/docs/configuration/interpolation.html#file_path_). Note: Must provide `private_key` when using this.
```

Comment:
This link and the one above do not match (I haven't checked which one is/will be correct).

Also remove the "Note:" -- It's a required part of the configuration. You can just state it directly, something like "The `certificate` argument must be used in conjunction with a `private_key`"?

Reply:
`/docs/configuration/functions/file.html` is the new location of that documentation for v0.12, so that's the correct one if this is going to merge after we land the v0.12 dev branch into master.